main.js

```diff
@@ -11,3 +20,25 @@ function addEvent() {
   scheduleText.textContent = 'Scheduled Events For ' + event.target.textContent
   console.log(event.target)
 }
+
+addEntryButton.addEventListener('click', modalAppear)
+
+function modalAppear() {
+  modal.style.display = 'initial'
+}
+
+submitButton.addEventListener('click', addText)
+
+function addText() {
+  modal.style.display = 'none'
+  for(var i = 0; i < timeData.length; i++) {
+    if(timeData[i].textContent === '') {
+      timeData[i].textContent = hours.value + ':' + minutes.value + ' ' + amPm.value
+      descriptionData[i].textContent = descriptionText.value
```

Comment:
I know in the wireframe it looks like we need to keep the rows filled, but when you are dynamically creating the data it is ok to have an empty table body and just build it as the user inputs data